composer.json

```diff
@@ -11,12 +11,11 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "ext-curl": "*",
-        "symfony/polyfill-php56": "^1.10"
+        "php": ">=7.2.0",
```

Comment:
:heart:

Reply:
:)

BTW from https://github.com/messagebird/php-rest-api/issues/79#issuecomment-499866063 this could be published as a minor version bump and let composer's php version resolution to avoid having users of unsupported versions upgrade.

Reply:
This should be updated in the README, it still says php >= 5.4